pages/common/git.md

```diff
@@ -1,7 +1,7 @@
 # git
 
 > Distributed version control system.
-> Some subcommands such as `git commit` have their own usage documentation.
+> Some subcommands such as `commit`, `checkout`, etc. have their own usage documentation. To see tldr for them use `tldr git {{subcommand}}`.
```

Comment:
Interesting idea, but I feel it's more explicit to say `git commit` instead of just `commit` for example.

Reply:
I'm not sure if we should change that part at all. I assume (and hope) that most users will know that this means you can run `tldr git commit`.

Reply:
But token syntax in description line isn't supported right?

Reply:
Oh yeah! We haven't done that by convention, no. Let's apply your suggestion as-is then.